src/R/Components/Impl/PackageManager/Implementation/RPackageManager.cs

```diff
@@ -60,79 +60,38 @@ internal class RPackageManager : IRPackageManager {
             }
```

Comment:
Yes, it should, but I haven't found `rtvs:::package_install`. Though I think it should be a separate change

---
In reply to: [73064953](https://github.com/Microsoft/RTVS/pull/2133#discussion_r73064953) [](ancestors = 73064953)